dev.yml

```diff
@@ -0,0 +1,12 @@
+---
+name: kubernetes-deploy
+up:
+  - ruby: 2.4.1
+  - bundler
+  - minikube
+
+minikube: {}
+
+commands:
+  test:
```

Comment:
Another command that could be really handy (I have a personal alias for it I use all the time) is something like:
```yaml
  tophat:
    run: PRINT_LOGS=1 bundle exec ruby -I test test/integration/kubernetes_deploy_test.rb -n/${1}/
    desc: Tophat a change by running a test scenario with logging output enabled.
    syntax:
      optional:
        argument: TEST_REGEX
```

![image](https://user-images.githubusercontent.com/4789493/28483325-aa6226c2-6e3a-11e7-966e-452cc81045bb.png)